app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -143,15 +142,28 @@ class EnabledContent extends ImmutableComponent {
 
   fundsAmount () {
     const ledgerData = this.props.ledgerData
-    const val = formatCurrentBalance(ledgerData) || ''
-    const big = val.length > 23
-
-    return <FormTextbox
-      readOnly
-      data-test-id='fundsAmount'
-      value={val}
-      customClass={big && styles.width_input}
-    />
+    if (!ledgerData) {
+      return
+    }
+
+    const total = formatCurrentBalance(ledgerData, ledgerData.get('balance'), false) || ''
+    const userFunded = formatCurrentBalance(ledgerData, ledgerData.get('userFunded')) || ''
+    const grants = ledgerData.get('grants') || Immutable.List()
+
+    return <div className={css(styles.fundsAmount)}>
+      <div className={css(styles.fundsAmount__item)}>{userFunded}</div>
+      {
+        grants.map(grant => {
+          return <div className={css(styles.fundsAmount__item)}>
+            {formatCurrentBalance(ledgerData, grant.get('amount'), false)}
+            <span> (<span data-l10n-id='expires' /> {new Date(grant.get('expirationDate') * 1000).toLocaleDateString()})</span>
```

Comment:
(nit) could we maybe separate `new Date(grant.get('expirationDate') * 1000).toLocaleDateString()` in a to a formatting function?

Reply:
Not sure if needed